plugin/src/main/java/io/jenkins/plugins/analysis/core/util/LogHandler.java

```diff
@@ -30,8 +32,8 @@ public class LogHandler {
      * @param name
      *         the name of the logger
      */
-    public LogHandler(final TaskListener listener, final String name) {
-        this(listener, name, 0, 0);
+    public LogHandler(final TaskListener listener, final String name, final boolean quiet) {
```

Comment:
I think we need to preserve the old constructors since this class is public and might be used somewhere else (so delegation is required)